aws-lc-rs/src/agreement.rs

```diff
@@ -307,13 +307,11 @@ impl PrivateKey {
                 )
             })?
         } else {
-            let ec_group = unsafe { ec_group_from_nid(alg.id.nid())? };
-            let private_bn = DetachableLcPtr::try_from(key_bytes)?;
+            let ec_group: LcPtr<EC_GROUP> = ec_group_from_nid(alg.id.nid())?;
```

Comment:
Minor but `ec_group_from_nid` doesn't technically need this since you aren't trying to convert to the type like the line right below it. Not really picky, just wasn't sure if you were doing it for clarity or had expected different.